backend/tests/test_hacknight.py

```diff
@@ -31,3 +34,80 @@ def test_get_hacknights_unauthorized(client, add_hacknights):
     response = rv.get_json()
     assert rv.status_code == HTTPStatus.UNAUTHORIZED
     assert response['msg'] == 'Missing Authorization Header'
+
+
+def test_add_participants_to_hacknight(
+    access_token, add_hacknights, add_participants, client
+):
+    """Test add new participant to hacknight."""
+    payload = {'participants': [1, 3]}
+    rv = client.patch(
+        '/hacknights/1/',
+        headers={'Authorization': 'Bearer {}'.format(access_token)},
+        data=json.dumps(payload)
+    )
+    resp = rv.get_json()
+    participants_ids = [
+        participant['id'] for participant in resp['hacknight']['participants']
+    ]
+    assert rv.status_code == HTTPStatus.OK
+    for participant in resp['hacknight']['participants']:
+        assert participant['id'] in payload['participants']
+
+
+def test_add_participants_to_hacknight_unauthorized(
+    add_hacknights, add_participants, client
+):
+    """Test add participants to hacknight when user is not logged in."""
+    payload = {'participants': [1, 3]}
+    rv = client.patch('/hacknights/1/', data=json.dumps(payload))
+    response = rv.get_json()
+    assert rv.status_code == HTTPStatus.UNAUTHORIZED
+    assert response['msg'] == 'Missing Authorization Header'
+
+
+def test_add_nonexistent_participants_to_hacknight(
+    access_token, add_hacknights, client
+):
+    """Test add non-existent participants ids to hacknight."""
+    payload = {'participants': [1, 3]}
+    rv = client.patch(
+        '/hacknights/1/',
+        headers={'Authorization': 'Bearer {}'.format(access_token)},
+        data=json.dumps(payload)
+    )
+    response = rv.get_json()
+    assert rv.status_code == HTTPStatus.NOT_FOUND
+
+
+def test_add_participants_to_nonexistent_hacknight(
+    access_token, add_participants, client
+):
+    """Test add participants to non-existent hacknight."""
+    payload = {'participants': [1, 3]}
+    rv = client.patch(
+        '/hacknights/1/',
+        headers={'Authorization': 'Bearer {}'.format(access_token)},
+        data=json.dumps(payload)
+    )
+    response = rv.get_json()
+    assert rv.status_code == HTTPStatus.NOT_FOUND
+
+
+def test_duplicate_participant_in_hacknight(
+    access_token, add_hacknights, add_participants, client, _db
+):
+    """Test add participant who is in hacknight already."""
```

Comment:
Add participant who is already in hacknight.

It's not perfect, but sounds better. ;-)